WikiWatershed/model-my-watershed
# Unit Conversion: HydroShare Export

## Overview

Augments HydroShare CSV Generation code to respect user's preferred unit scheme.

Connects #3028 

## Testing Instructions

* Check out this branch
* Set your scheme to US Customary
* Export to HydroShare
* Download result CSVs from HydroShare
* Ensure they are in the correct units
